cross-files/armv7a-linux-android

```diff
@@ -1,9 +1,12 @@
 
 # Tested with Android NDK r18, standalone toolchain
-# Targeting API level 24
+# Targeting API level 21
+#
+# When targeting API levels < 24 must undef _FILE_OFFSET_BITS to addess this issue:
```

Comment:
Maybe also add to the comment that _FILE_OFFSET_BITS is defined by meson (and also a link to <https://github.com/mesonbuild/meson/pull/2996#issuecomment-384045808>) to help explain exactly what the issue is? I'm sure I'll forget the details in a few weeks.